be/src/vec/functions/function_string.h

```diff
@@ -294,6 +294,72 @@ struct SubstringUtil {
     }
 };
 
+class FunctionStrcmp : public IFunction {
+public:
+    static constexpr auto name = "strcmp";
+
+    static FunctionPtr create() { return std::make_shared<FunctionStrcmp>(); }
+
+    String get_name() const override { return name; }
+
+    size_t get_number_of_arguments() const override { return 2; }
+
+    DataTypePtr get_return_type_impl(const DataTypes& arguments) const override {
+        if (arguments[0]->is_nullable() || arguments[1]->is_nullable()) {
```

Comment:
no need consider nulllable in `use_default_null`

Reply:
fixed